src/main/java/com/example/balanceGame/service/BoardService.java

```diff
@@ -52,6 +55,19 @@ public ResponseEntity<BoardDetailResponse> findBoardProfile(long boardKey) {
         return new ResponseEntity<>(boardDetailResponse, HttpStatus.OK);
     }
 
+    public ResponseEntity<FindAllByDateResponse> findAllByDate(PageRequest pageRequest) {
```

Comment:
```suggestion
    public PageResult<BoardDTO> findAllByDate(PageRequest pageRequest) {
```

service 레이어에서 http 관련 의존성을 가지는 건 지양하는게 좋음

별도 DTO 만들어서 return 하고 그에 따른 Response 매핑은 controller 단에서 하는게 좋음

보통 Layered Architecture 라고 하는 개념을 사용하는데, 보통은 아래처럼 나누는데 하단 계층이 상단 계층을 침범하면 안됨.

왜냐면 만약 지금은 Controller 에서  REST 방식으로 제공한다고 한들 이게 나중에 XML, GRPC 등으로 제공해야하는데 HTTP 의존성을 Domain or Application 계층에서 사용하면 나중에 수정하기가 빡셈

 각자의 역할에 맞는 행위를 하는게 좋음

지금도 Domain 계층에서 Presentation 의존성을 가져 버리면 안되니까 (반대로 Presentation 은 Domain 보다 상위 계층이니 Domain 의 의존성은 가져도 됨)

- Presentation (controller, http 관련, network 관련, ui 관련)
- Application (service, Facade, 각종 도메인 메소드 집합하여 사용)
- Domain (Service, 비즈니스 로직)
- Data or Persistence (Enity, Repository, 영속성 관련 로직)